WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagViewController.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+final class SiteTagViewController: UITableViewController {
+    private let blog: Blog
+    private let tagsService: PostTagService
+    private let tag: PostTag
+    
+    public init(blog: Blog, tag: PostTag, tagsService: PostTagService) {
```

Comment:
Just a quick note about the way `Service` classes are intended to be used. :)
The thought behind service classes is that they should be short lived, stateless (mostly), and disposable.  It would be very unusual for one controller to instantiate a service and then pass that service to another controller to use, or to store the instance in a property.

Reply:
Understood. My intention by passing it around this way was "injecting" it, for better testability. I will make the Details view instantiate it directly, just when it needs it (and not retain it as a property)